misc/custom-flows-environment-variables/subdir/use-env-variables.sh

```diff
@@ -0,0 +1,2 @@
+echo user: $USER_NAME
+echo user email: $USER_EMAIL
```

Comment:
why we need both `use-env-variables.sh and `subdir/use-env-variables.sh`? 
also `print-user-id.sh` is reduandant IMO. 

we can just replace those 3 files with 1 that prints all 3 variables.

Reply:
739dc2ee

kept usages of scripts and the plain usage to demonstrate both ways are valid. removed subdir.